tests/warp/warp_test.go

```diff
@@ -39,9 +43,20 @@ import (
 const fundedKeyStr = "56289e99c94b6912bfc12adc093c9b51124f0dc54ac7a766b2bc5ccf558d8027"
 
 var (
-	config              = runner.NewDefaultANRConfig()
-	manager             = runner.NewNetworkManager(config)
-	warpChainConfigPath string
+	config                         = runner.NewDefaultANRConfig()
+	manager                        = runner.NewNetworkManager(config)
+	warpChainConfigPath            string
+	unsignedWarpMsg                *avalancheWarp.UnsignedMessage
+	unsignedWarpMessageID          ids.ID
+	signedWarpMsg                  *avalancheWarp.Message
+	blockchainIDA, blockchainIDB   ids.ID
+	chainAURIs, chainBURIs         []string
+	chainAWSClient, chainBWSClient ethclient.Client
+	chainID                        = big.NewInt(99999)
+	fundedKey                      *ecdsa.PrivateKey
+	fundedAddress                  common.Address
+	payload                        = []byte{1, 2, 3}
+	txSigner                       = types.LatestSignerForChainID(chainID)
```

Comment:
Does moving around these variables need to be part of this PR? If not, could we separate it out?

Reply:
https://github.com/ava-labs/subnet-evm/pull/940/files